spyder/app/tests/test_mainwindow.py

```diff
@@ -801,6 +802,7 @@ def test_runconfig_workdir(main_window, qtbot, tmpdir):
     sys.platform.startswith("linux") and running_in_ci(),
     reason='Fails sometimes on Linux and CIs'
 )
+# @pytest.mark.skipif(PYQT6, reason="Fails with PyQt6")
```

Comment:
Please remove this comment because it's not necessary.

Reply:
Done